build-tools/packages/build-cli/src/BasePackageCommand.ts

```diff
@@ -127,31 +173,70 @@ export abstract class PackageCommand<
 		}
 	}
 
+	/**
+	 * Runs processPackage for each package in a release group, exlcuding the root.
+	 */
+	private async processReleaseGroup(
+		releaseGroup: ReleaseGroup,
+		rootPackageOnly: boolean,
+	): Promise<void> {
+		this.info(`Finding packages for release group: ${releaseGroup}`);
+		const ctx = await this.getContext();
+		if (rootPackageOnly) {
+			const rg = ctx.repo.releaseGroups.get(releaseGroup);
+			assert(rg !== undefined);
+			return this.processPackages([rg.repoPath], { kind: "releaseGroupRoot" });
+		}
+
+		return this.processPackages(
+			ctx.packagesInReleaseGroup(releaseGroup).map((p) => p.directory),
+			{ kind: "releaseGroupPackage" },
+		);
+	}
+
+	/**
+	 * Runs processPackage for each independent package in the repo.
+	 */
+	private async processIndependentPackages(): Promise<void> {
```

Comment:
Same as above: I think having this return the list of found packages would be better.